checks/org/check.go

```diff
@@ -34,6 +39,10 @@ func (o *check) Check(ctx context.Context, i voucher.ImageData) (bool, error) {
 		return false, err
 	}
 
+	if o.repositoryClient == nil {
+		return false, errors.New("unable to connect to " + buildDetail.RepositoryURL)
```

Comment:
Could we change this to a constant error?

```golang
var ErrNoRepositoryClient = errors.New("no repository client configured for check")
```

```suggestion
		return false, ErrNoRepositoryClient
```